### PR TITLE
Feature responsive navbar

### PR DESCRIPTION
**Pull request by Dave00913**

### CODE REVIEW

**Review by ohana-matsumae — APPROVED** [review]
very cool

**Review by candace1818 — APPROVED** [review]
amazing

File: SpotlessSolutions.Web/ClientApp/src/components/navigation/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  return (
+    <>
+      <nav className="bg-[#05132F] fixed w-full z-20 top-0 start-0 border-b border-gray-200 dark:border-gray-600 shadow-md rounded-b-md">
+        <div className="max-w-screen-xl flex flex-wrap items-center justify-between mx-auto p-6">
+          <a
+            href=""
+            className="flex items-center space-x-3 rtl:space-x-reverse"
+          >
+            <img
+              src="src/assets/td_logo.jpg"
```
Comment (ohana-matsumae) [inline SpotlessSolutions.Web/ClientApp/src/components/navigation/Navbar.tsx:20]: Use imports

**Review by ohana-matsumae — CHANGES_REQUESTED** [review]

**Review by ohana-matsumae — APPROVED** [review]

**Comment by ohana-matsumae** [comment]
Use the latest branch first and then make your changes there with related changes. This PR will be closed.

**Pull request closed without merging**